cli/lib/command/internal_command_commit.dart

```diff
@@ -74,23 +75,30 @@ class InternalCommandCommit extends InternalCommand {
       originalBranchName = args[1];
     }
     final resultingBranchName = sanitizeBranchName(originalBranchName);
-    if (!acceptBranchName && originalBranchName != resultingBranchName) {
+
+    // Add branch prefix if configured
+    final branchPrefix = Settings.instance.settings['branch_prefix'];
+    final prefixedBranchName = branchPrefix != null && branchPrefix.isNotEmpty
```

Comment:
You should have control over `branchPrefix` setting, you can configure it in a way that would make any `null` into empty string, so you would not need to check for this values here.